.travis.yml

```diff
@@ -17,7 +17,7 @@ env:
     - DIR=transmart-rest-api; TEST=check
     - DIR=transmart-core-db; INSTALL=":transmart-data:createVars :transmart-data:setupPostgresTest"; TEST=check
     - DIR=transmart-api-server; TEST=check
-    - DIR=transmart-copy; INSTALL=":transmart-data:createVars :transmart-data:setupPostgres"; PREPARE=shadowJar; TEST=check
+    - DIR=transmart-copy; INSTALL=":prepareTestDatabase"; PREPARE=shadowJar; TEST=check
```

Comment:
looks like the `before_install` part can be also cleaned up a bit now, e.g. php and gnupg2 checks could be removed, right?

Reply:
https://github.com/thehyve/transmart-core/blob/transmart-essentials/.travis.yml#L8

Reply:
I am not sure if those can already be removed. I think php still plays a role in bundling the transmart-data artefact, when `gradle publishToMavenLocal` is run.